### ISSUE TITLE
Segfault when overwriting files

### ISSUE THREAD

**Issue by eminence** [tunabrain/tungsten#9]
Try setting `"overwrite_output_files": true` 

I get this traceback:

```
#0  0x00007ffff77715b9 in std::string::assign(std::string const&) () from /usr/lib/gcc/x86_64-pc-linux-gnu/4.8.3/libstdc++.so.6
#1  0x00000000006422eb in Tungsten::Path::operator= (this=0x18) at /storage/home/achin/devel/tungsten/src/core/io/Path.hpp:15
#2  0x0000000000678279 in Tungsten::FileUtils::openOutputStream (p=...) at /storage/home/achin/devel/tungsten/src/core/io/FileUtils.cpp:419
#3  0x0000000000730fbb in Tungsten::ImageIO::savePng (path=..., 
```

It looks like `auto iter = _metaData.find(out.get());` seems to be returning something it shouldn't.

**Comment by tunabrain** [comment]
It looks like I forgot to update the Linux branch in an #ifdef _WIN32 section inside FileUtils.

I just pushed a fix. Can you confirm that it works?

**Comment by eminence** [comment]
Fixed by ca85b8d92fb6f3230aaea40d172100bfcb0c22a6
